website/content/en/docs/main/tutorials/bookinfo/index.md

```diff
@@ -20,17 +20,7 @@ System illustration:
 
 ## Install ClusterLink CLI
 
-1. Install ClusterLink CLI on Linux or Mac using the installation script:
-
-    ```sh
-    curl -L https://github.com/clusterlink-net/clusterlink/releases/latest/download/clusterlink.sh | sh -
-    ```
-
-1. Verify the installation:
-
-    ```sh
-    clusterlink --version
-    ```
+{{% readfile file="/static/files/tutorials/cli-installation.md" %}}
```

Comment:
nit/fyi: not sure if the full path is needed - maybe `/static` is referenced implicitly. But it works just the same.